### PR TITLE
Update api document for a API

### PR DESCRIPTION
**Pull request by irham0019**

### CODE REVIEW

File: components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-client-js/public/js/lib/api.js [hunk 1]
```diff
@@ -656,15 +656,37 @@ class API {
             if (callback) {
                 return promise_get_all.catch(AuthClient.unauthorizedErrorHandler).then(callback);
             } else {
-                return promise_get_all;
+                return promise_get_all;s
             }
```
Comment (pubudu538) [inline components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-client-js/public/js/lib/api.js]: Let's remove s here.

File: components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-details/public/js/details.js [hunk 2]
```diff
@@ -579,13 +579,19 @@ function initDataTable(raw_data) {
             callback(raw_data);
         },
         columns: [
+            {'data': 'documentId'},
             {'data': 'name'},
-            {'data': 'type'},
-            {'data': null},
+            {'data': 'sourceType'},
+ //           {'data': null},
             {'data': null},
```
Comment (pubudu538) [inline components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-details/public/js/details.js]: Let's not add commented code here. We can use 'to do' if needed.

File: components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-documents/api-documents.hbs [hunk 3]
```diff
@@ -4,8 +4,9 @@
         <div class="control-group">
             <input type="hidden" id="docAPIVersion" value="1.0"/>
             <input type="hidden" id="docAPIName" value="test"/>
-            <div id="addDoc">
-                <a id="add-new-doc" href="#" title="Add New Document" class="btn btn-primary add-new-doc btn-sm padding-reduce-on-grid-view">
+            <input type="hidden" id="docId" value="test"/>
+            <div id="doc-header">
+                <a id="add-new-doc" href="#" title="Add New Document" onclick="javascript:$('#add-doc-submit').show();$('#update-doc-submit').hide();" class="btn btn-primary add-new-doc btn-sm padding-reduce-on-grid-view">
                     <i class=" glyphicon glyphicon-plus-sign" title="Add New Document"></i>
```
Comment (pubudu538) [inline components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-documents/api-documents.hbs]: We can add language support for Add New Document.

File: components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-client-js/public/js/lib/api.js [hunk 4]
```diff
@@ -656,15 +656,37 @@ class API {
             if (callback) {
                 return promise_get_all.catch(AuthClient.unauthorizedErrorHandler).then(callback);
             } else {
-                return promise_get_all;
+                return promise_get_all;s
```
Comment (tharikaGitHub) [inline components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-client-js/public/js/lib/api.js]: Can we delete the 's' at the end of the line?

File: components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-documents/api-documents.hbs [hunk 5]
```diff
@@ -149,9 +109,10 @@
     <table class="table table-striped table-hover table-bordered display data-table" id="doc-table">
         <thead>
         <tr>
+            <th class="doc-listing-Id">Id</th>
             <th class="doc-listing-name">Name</th>
-            <th>{{i18n "Type"}}</th>
-            <th>{{i18n "Modified On"}}</th>
+            <th>{{i18n "Source"}}</th>
+            <!--<th>{{i18n "Modified On"}}</th>-->
```
Comment (anuruddhal) [inline components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-documents/api-documents.hbs]: Shall we remove this line?

**Pull request merged**